### DISCUSSION TITLE
Twilight Forest Uncrafting Table is disabled, is this intended?

### DISCUSSION THREAD

**Discussion by mjevans** [EnigmaticaModpacks/Enigmatica8#249] [Q&A]
https://github.com/EnigmaticaModpacks/Enigmatica8/blob/4dbd01427fd0ab5b74966aa8d7ab43e313fc667b/config/twilightforest-common.toml#L69

The Twilight Forest uncrafting table is disabled.  This seems counter-intuitive at a glance since some items already spawn tagged with "twilightforest:banned_uncrafting_ingredients".  A search for $uncrafting makes it more clear these were added on Twilight Forest items (likely by the mod rather than the pack).

Is it intended for the uncrafting table to be banned?  Is this until there's time to add the "twilightforest:banned_uncrafting_ingredients" tag to items that should not be uncrafted or permanent?

**Comment by NielsPilgaard** [comment]
It is indeed intended, it's too much work to find and disable all uncrafting recipes that dupe 😞
